model.py

```diff
@@ -0,0 +1,30 @@
+from mesa import Model
+from mesa.time import RandomActivation
+from agent import Person
+from mesa.datacollection import DataCollector
+
+
+
+class Model(Model):
+    def __init__(self, NUM_PEOPLE, INITIAL_MONEY, 
+                 SPENDING_PROB, SPENDING_AMOUNT, SALARY):
```

Comment:
I would set the default values here.
`def __init__(self, num_people=10, initial_money=1000, spending_prob=0.5, spending_amount=100, salary=1000)`